src/types.h

```diff
@@ -5,19 +5,20 @@
 #include "os.h"
 #include "inline_keywords.h" /* For H_INLINE */
 #include <stddef.h>
+#include <stdint.h>
 
 /* Some generic, cross-platform types. */
 
 struct _MMPoint {
-	size_t x;
-	size_t y;
+	int64_t x;
+	int64_t y;
```

Comment:
I think it's good that we're moving away from compiler-specific types especially in cross platform/interface structs. :+1: